insecure/attackOrchestrator.go

```diff
@@ -2,12 +2,17 @@ package insecure
 
```

Comment:
please use snake_case for all file names

Reply:
renamed files from camelCase to snake_case

https://github.com/onflow/flow-go/pull/2936/commits/f0ad8fc12c0c45c10bdbd04f73ba8a09c165b451